contracts/HubPool.sol

```diff
@@ -188,11 +188,6 @@ contract HubPool is Testable, Lockable, MultiCaller, Ownable {
     ) public onlyOwner {
         //TODO In the future this should call cross-chain adapters to setEnableRoute.
         whitelistedRoutes[originToken][destinationChainId] = destinationToken;
-
-        emit WhitelistRoute(destinationChainId, originToken, destinationToken);
-
-        // TODO: Should relay message to L2 for destinationChainId and call setEnableRoute(originToken, destinationChainId, true)
```

Comment:
why remove comment? is this TODO resolved?

Reply:
mistake in merging. added back.